Makefile

```diff
@@ -23,4 +23,7 @@ build:
 
 test:
 	@echo -- $@ --
-	go test ./... -v -cover -coverprofile analyzer.coverprofile
+	 go test ./... -v -coverpkg=./... -coverprofile analyzer.coverprofile
```

Comment:
Maybe define a variable to contain `analyzer.coverprofile` and use it in the two directives

Reply:
done